src/schema/rules/sidecars/mrs.yaml

```diff
@@ -0,0 +1,98 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+# MRS Common metadata fields
+MRSScannerHardware:
+  selectors:
+    - modality == "mrs"
+  fields:
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    InstitutionalDepartmentName: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    DeviceSerialNumber: recommended
+    StationName: recommended
+    SoftwareVersions: recommended
+    MagneticFieldStrength: recommended
+    ReceiveCoilName: recommended
+    ReceiveCoilActiveElements: recommended
+    NumberReceiveCoilActiveElements: optional
+    NumberTransmitCoilActiveElements: optional
+
+MRSSample:
+  selectors:
+    - modality == "mrs"
+  fields:
+    BodyPart:
+      level: optional
+      description_addendum: Corresponds to DICOM Tag 0018, 0015 `Body Part Examined`.
+    BodyPartDetails: optional
+    BodyPartDetailsOntology: optional
+
+MRSSequenceSpecifics:
+  selectors:
+    - modality == "mrs"
+  fields:
+    PulseSequenceType:
+      level: recommended
+      description_addendum: |
+        For example, `"sLASER"`, `"MEGA-PRESS"`, `"EPSI"`, `"Metabolite-cycled MRSI"`.
+    ScanningSequence__mrs: recommended
+    SequenceName: recommended
+    PulseSequenceDetails: recommended
+    MRAcquisitionType: recommended for MRSI
+    WaterSuppression: recommended
+    WaterSuppressionTechnique: optional
+    OuterVolumeSuppression: recommended
+    B0ShimmingTechnique: optional
+    B1ShimmingTechnique: optional
+
+MRSRequiredFields:
+  selectors:
+    - modality == "mrs"
+  fields:
+    ResonantNucleus: required
+    SpectrometerFrequency: required
+    SpectralWidth: required
+    EchoTime: required
+
+MRSRecommendedFields:
+  selectors:
+    - modality == "mrs"
+  fields:
+    NumberOfSpectralPoints: recommended
+    NumberOfTransients:
+      level: optional
+      level_addendum: recommended for SVS and unloc
+    RepetitionTime__mrs: recommended
+    MixingTime: recommended
+    InversionTime: recommended for inversion recovery data
+    FlipAngle: recommended
+    AcquisitionVoxelSize: recommended
+    MatrixSize: recommended for MRSI
+    VolumeAffineMatrix: recommended for MRSI
+    EditTarget: recommended for edited data
+    EditPulse: recommended for edited data
+    EditCondition: recommended for edited data
+    EncodingTechnique: recommended for MRSI
+    ReferenceSignal: recommended if MRS reference data are present
```

Comment:
Q: Would there be a single reference signal for many `svs/mrsi/unloc` scans? If not, possibly this could be left as an association, where `mrsref` with all the same entities applies?

I'm trying to think how to validate this, and checking for all of the suffixes present in the dataset is a bit more of a pain than checking for all of the datatypes or modalities.

Reply:
Do you mean for a subject that has, in theory, `svs`, `mrsi`, and `unloc` scans, would a single reference apply to them all? If so, no, because the reference has to be specific to the acquisition. So, each of the three would have to have their own references.

If you mean instead if, say, a subject has repeat `svs` (or `mrsi/unloc`) scans (e.g., two scans for test-retest), would a single reference apply to them both? Then, that is more of a study design decision. Personally, I would acquire a separate reference signal for each individual `svs/mrsi/unloc` scan I'd acquire.

Reply:
Okay, so it may frequently be, but is not always, 1-1. I'll mull on this a little.

Reply:
I have not been able to think of a way in the current schema to make this reliably produce a warning when there are `mrsref` present but not when there are none. As written, it will simply always warn, which would be no more than annoying to someone who didn't collect reference scans.

I think this is fine.